src/ProjectService.Validation/File/FileContentValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.Kernel.Validators.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Dto.Models;
+using LT.DigitalOffice.ProjectService.Validation.File.Interfaces;
+
+namespace LT.DigitalOffice.ProjectService.Validation.File.CreateFileValidator
+{
+  public class FileContentValidator : AbstractValidator<FileContent>, IFileContentValidator
+  {
+    public FileContentValidator(
+      IImageContentValidator contentValidator)
+    {
+      RuleFor(i => i.Content)
```

Comment:
1) Why do you use an image content validator for the file?
2) And should a service that does not work with an entity validate it? ((If validation changes, then you will have to change all the services, instead of the only FIle that is responsible for this)

Reply:
done